adrs/8618797096/README.md

```diff
@@ -0,0 +1,61 @@
+# Usage of JSX Fragment
+
+* **State:** Draft
+* **Created:** 2020-11-15
+* **Tags:** jsx
+
+## Context
+
+Today we have a way to specify fragments in our React, Preact codebase or
+any other library that adopts such technology.
+
+1. Using `Fragment` from React import:
+
+    ```tsx
+    import { Fragment } from 'react';
+    function MyApp() {
+      return (
+        <Fragment></Fragment>
```

Comment:
The best arguments I've heard for this syntax:

1. Beginners might not recognize `<></>` and you can't google a bunch of symbols
    ![Screen Shot 2020-11-15 at 11 34 15 AM](https://user-images.githubusercontent.com/5052422/99194783-936c5600-2736-11eb-9186-4855f6581691.png)
    (Yes, this is contrived, newbie googling skills)
2. A codebase could have tons of existing `<Fragments />`, and for a good reason, so adopting this ADR would require touching existing code.   

    The tooling to support the fragment syntax trailed the release of the Fragment element.    At the time of release, Babel's support was still in Beta. And anecdotally, that timing gap was widened because of upstream dependencies, such as `create-react-app`.

    See https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html#babel.

Reply:
> Beginners might not recognize <></> and you can't google a bunch of symbols

Yeah, although I totally understand the concerns, learning JSX requires you to get familiar and understand basic concepts.

> A codebase could have tons of existing <Fragments />, and for a good reason, so adopting this ADR would require touching existing code.

Worth articulating that ADRs don't imply that work will no be needed, 🤷🏻 it is what it is, ADR provides records about the decision made and consensus, but it is up to you to adopt them.